wormhole-connect/src/utils/transferValidation.ts

```diff
@@ -99,42 +99,8 @@ export const validateAmount = (
   return '';
 };
 
-async function checkAddressIsSanctioned(address: string): Promise<boolean> {
-  if (trmCache[address]) {
-    return trmCache[address].isSanctioned;
-  }
-  const defaultAuth =
-    'Basic ' + Buffer.from('<username>:<password>').toString('base64');
-  const res = await fetch(
-    `https://api.trmlabs.com/public/v1/sanctions/screening`,
-    {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json',
-        Authorization:
-          process.env.REACT_APP_TRM_API_KEY && WH_CONFIG.env === 'MAINNET'
-            ? process.env.REACT_APP_TRM_API_KEY
-            : defaultAuth,
-      },
-      body: JSON.stringify([{ address }]),
-    },
-  );
-
-  if (res.status !== 200 && res.status !== 201) {
-    // set cache so it stops making requests
-    if (res.status === 429) {
-      trmCache[address] = {
-        address,
-        isSanctioned: false,
-      };
-    }
-    return false;
-  }
-
-  const data = await res.json();
-  trmCache[address] = data[0];
-  return data[0].isSanctioned;
-}
+const checkAddressIsSanctioned = (address: string): boolean =>
+  SANCTIONED_WALLETS.has(address) || SANCTIONED_WALLETS.has('0x' + address);
```

Comment:
I think the address would already be prefixed with `0x`, unless it's a Solana address or doesn't use that formatting

Reply:
Imo better to not pass that assumption to the caller, and just check it here.